src/pages/guides/index.md

```diff
@@ -235,17 +235,64 @@ For Creative Cloud Asset events, you'll need to add the Creative Cloud Libraries
+
+- sending additional field of recipient_client_id as part of your event payload
+- signing the event payloads digitally and sending those digital signatures as request headers to your webhook
+
+I/O Events generates two public-private key pairs and sign your event payload using digital signatures following the steps below
+
+- a message digest of your event payload is computed by applying `rsa-sha256` hash function algorithm
+- the digest is then encrypted using the I/O Events private key to generate the digital signature
+
+I/O Events sends the 2 digital signatures as webhook request headers and they are available via the header fields
+`x-adobe-digital-signature-1` and `x-adobe-digital-signature-2` respectively. 
+
+I/O Events also sends 2 public keys corresponding to the private keys used to generate the digital signatures. These public keys which are stored in Cloud Front and are publicly accessible using the Cloud Front urls which look like `https://xxxxx.cloudfront.net/pub-key-<random-uuid>.pem`. I/O Events sends the public keys urls for both the keys and they are available via the the request header fields `x-adobe-public-key1-url` and  `x-adobe-public-key2-url` respectively.
+
+As mentioned earlier, I/O Events transforms your event payload and adds an additional json field `recipient_client_id` to your payload. See the sample payload after the transformation that I/O Events sends to your webhook.
+
+![Sample XDM format asset event payload](./img/xdm_asset_payload_with_recipient_clientid.png "Sample XDM format asset event payload")
+
+Upon receiving a request, you must do the below for leveraging the enhanced security measures
+
+- verify you are the actual recipient of the event using the new `recipient_client_id` field available in the payload
+- once verified, your app should fetch the public key using the cloud front url present in the header
+- after downloading the public key set it in the cache with cache expiry of `not more than 24h` 
+
+You can also consider implementing a retry mechanism to call public key urls in case of any transient error that might occur.  
+
+Once you have the public keys fetched as plain text you can now verify the digital signatures by following the steps as below
+
+- decrypt the message digest using the public key
+- compute the hash message digest of the event payload (available in the webhook request body) using the same hash function algorithm `rsa-sha256` used by I/O Events during signing
+- now validate each signature by comparing 
+  - the message digest computed by hashing 
+  - and the digest received after decrypting the signature using the public key
+
+Refer to [this](https://github.com/adobe/aio-lib-events/blob/4694ef1b4f19c63131d506e1abfe7a3e555e2914/src/index.js#L491) signature verification method of the events sdk to understand the above signature validation steps for your webhook app.
```

Comment:
We would still need to change this link right?

Reply:
yes, once the sdk changes gets merged, will use the link and then merge

Reply:
ok then let us know when the link is updated and we'll review-again/approve

Reply:
updated.